docker-compose-mongo-replica-local.yml

```diff
@@ -14,8 +14,8 @@ services:
       retries: 20
     environment:
       - PMM_DEBUG=1
-      - PERCONA_TEST_SAAS_HOST=check-dev.percona.com:443
-      - PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+      - PERCONA_TEST_PLATFORM_ADDRESS=https://check-dev.percona.com:443
+      - PERCONA_TEST_PLATFORM_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
```

Comment:
@ritbl We are not reached consensus about name for that variable, so I left it like this.